.yamato/package-test.yml

```diff
@@ -13,7 +13,11 @@ test_{{ platform.name }}_{{ editor.version }}:
     UPMCI_PKG: "upm-ci-utils@stable"
   commands:
       - npm install {% if platform.name == "win" %}"%UPMCI_PKG%"{% else %}"$UPMCI_PKG"{% endif %} -g --registry {{ registry.npm }}
-      - upm-ci package test -u {{ editor.version }} --package-path package/com.unity.formats.usd --type package-tests --extra-utr-arg="--api-profile=NET_4_6"
+      {% if editor.version == "trunk" %}
+        - upm-ci package test -u {{ editor.version }} --package-path package/com.unity.formats.usd --type package-tests --extra-utr-arg="--api-profile=NET_4_6" --extra-editor-arg=-editorAnalyticsTestMode        
```

Comment:
This is behind an if statement as there was a bug in the analytics side where if this was set for versions less than trunk, it would crash the editor

Reply:
Yeah I think that will have to be changed in the future, once neutron stuff becomes the new trunk (iirc 2023 is the last "classic" unity before neutron)

If there is a 2023.4 of "classic" Unity that becomes the new trunk, in that case we will have to add 2023.3 as part of this if statement